CMakeLists.txt

```diff
@@ -171,9 +171,7 @@ add_subdirectory(fdbbackup)
 add_subdirectory(contrib)
 add_subdirectory(tests)
 add_subdirectory(flowbench EXCLUDE_FROM_ALL)
-if(WITH_PYTHON)
-  add_subdirectory(bindings)
-endif()
+add_subdirectory(bindings)
```

Comment:
All bindings depend on python. We use a python script to generate some assembly code that is required for the C bindings

Reply:
Thanks for the tips :+1: